crates/matrix-sdk-crypto/src/identities/device.rs

```diff
@@ -764,6 +757,18 @@ impl ReadOnlyDevice {
         )
     }
 
+    pub(crate) fn is_cross_signed_by_owner(
+        &self,
+        device_owner_identity: &ReadOnlyUserIdentities,
+    ) -> bool {
+        match device_owner_identity {
+            ReadOnlyUserIdentities::Own(identity) => identity.is_device_signed(self).is_ok(),
```

Comment:
Could you please preserve the comments from the other body of this function?

Reply:
done